ra/ra.go

```diff
@@ -1837,11 +1837,10 @@ func (ra *RegistrationAuthorityImpl) revokeCertificate(ctx context.Context, seri
 	return nil
 }
 
-// updateRevocationForKeyCompromise generates a revoked OCSP response for the
-// already-revoked certificate with the given serial and issuer, and stores that
-// response in the database. This only works for certificates that were
-// previously revoked for a reason other than keyCompromise, and which are now
-// being updated to keyCompromise instead.
+// revokeCertificate updates the database to mark the certificate as revoked,
```

Comment:
The function name this doc stirng for is `updateRevocationForKeyCompromise`, not `revokeCertificate`.

Reply:
lmao, that's what I get for copy-pasting my comment edit. Thanks